cmd/controlloop/controlloop.go

```diff
@@ -66,7 +68,7 @@ func main() {
 	defer networkController.Shutdown()
 
 	s := gocron.NewScheduler(time.UTC)
-	schedule := cronExpressionFromFlatFile()
+	schedule := determineCronExpression()
```

Comment:
hm, I think you still need to react to updates to the cron expression; meaning, this will only occur when the process starts.

Either you use an higher lib - like [fsnotify](https://github.com/fsnotify/fsnotify) - or implement something like that using a goroutine that checks if the file's contents have changed, and if so, restart the scheduler.

I guess even exiting the process could be good enough (but worse than reloading the config), since the pod would restart.